### DISCUSSION TITLE
parser / syntax field not supported in custom postcss configuration?

### DISCUSSION THREAD

**Discussion by teehemkay** [vercel/next.js#23255] [Help]
Hello,

We're using the `postcss-scss` parser to enable parsing `.scss` files. It usually works perfectly.
But with the [`parser`](https://github.com/postcss/postcss-scss#2-inline-comments-for-postcss) (or the [`syntax`](https://github.com/postcss/postcss-scss#1-scss-transformations)) field set in `postcss.config.js`, `nextjs` complains with the following warning and our `.scss` files are not correctly processed:

```
Warning: Your PostCSS configuration defines a field which is not supported (`parser`). Please remove this configuration value.
```

Is there a way to setup a specific parser for `postcss`?

TIA

**Comment by philwolstenholme** [comment]
I just ran into this myself.

Looks like we can only change the `plugins` part of the config, not anything else: https://github.com/vercel/next.js/blob/canary/packages/next/src/build/webpack/config/blocks/css/plugins.ts#L139-L148

**Reply by pczern** [reply]
I would love to use https://github.com/postcss/sugarss, because it will prevent parsing of CSS through multiple tools by using only postcss instead of sass and postcss. Since I am using Tailwind, I already have postcss. Will the `parser` field be available in the future?
